### PR TITLE
Adds new landing page and updates internal page styling

### PR DESCRIPTION
**Pull request by alexabbott**
This pull request revises the landing page code to align with new designs.

This pull request also updates the internal pages to ensure consistent navigation and styling with the landing page.

### CODE REVIEW

File: theme/css/colors.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  --border-grey: #DCDCDC;
+  --border-red: #A80000;
+
+  /* Shadow Colors */
+  --shadow-grey: #9C9C9C;
+
+  /* Text Colors */
+  --text-black: #292321;
+  --text-red: #A30000;
+}
```
Comment (mattdangerw) [inline theme/css/colors.css:27]: lots of missing newlines at end of files.

File: theme/icons/.DS_Store
Comment (mattdangerw) [inline theme/icons/.DS_Store:1]: remove all `.DS_Store` files

File: theme/js/index.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  mobileNavControls.style.display = 'flex';
+  mobileNavSearchForm.style.display = 'none';
+});
+
+mobileNavMenu.addEventListener('click', () => {
+  mobileNavControls.style.display = 'flex';
+  mobileNavSearchForm.style.display = 'none';
+});
+
+// Parallax functionality
```
Comment (mattdangerw) [inline theme/js/index.js:68]: I wonder if this whole code block would be less jerky if you dynamically updated the element css to switch between position fixed and relative--so the render engine will keep fixed rather than trying to dynamically set a height in js. Not a web dev expert though!

File: theme/landing.html [hunk 3]
```diff
@@ -1,259 +1,686 @@
...
+        <div class="explore stacked__item">
+          <div class="explore__wrapper">
+            <div class="explore__column">
+              <div class="code__container">
+                <button class="code__copy--button">
+                  <i class="icon--copy"></i>
+                </button>
+                <pre>
+                  <code class="language-python">
+    inputs = keras.Input(shape=(32, 32, 3))
```
Comment (mattdangerw) [inline theme/landing.html:205]: We should remove the tab from these pre blocks so they copy without extra spaces. Instead just style the code card padding to get whatever left alignment we want.

**Review by mattdangerw — CHANGES_REQUESTED** [review]
Most of the comments are on that tracking sheet, but adding some comments here that are more code specific as well!
